develop/TOOLS/ThirdPartyTools/WiresharkDecryptTraffic.md

```diff
@@ -0,0 +1,28 @@
+---
+uid: WiresharkDecrypt
+---
+
+# Wireshark stream decryption
```

Comment:
It would be better if the title here and the title in the toc.yml were the same. Which is more appropriate, "Wireshark stream decryption" or "Wireshark package decryption"?

Reply:
Agreed, I would opt for "Wireshark stream decryption"